client/src/app/my-projects/my-projects.component.ts

```diff
@@ -33,7 +33,7 @@ export class MyProjectsComponent implements OnInit, OnDestroy {
     // protected against null user by the user-is-logged-in guard
     this.accountDataSource =  new SimpleDataSource(this.ingestService.getUserAccount.bind(this.ingestService));
     this.projectsDataSource = new PaginatedDataSource(params => {
-      return this.ingestService.getUserProjects({ ...params, sort: `${params.sort.column},${params.sort.direction}`})
+      return this.ingestService.getUserProjects({ ...params, sort: `${params.sort.column},${params.sort.direction}`});
```

Comment:
Semicolons in JavaScript are optional due to Automatic Semicolon Insertion (ASI)

Reply:
It's a style thing. You should either use them or not and it seems that we use them